spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -81,147 +71,586 @@ public class StreamApplicationTests {
-`spring.cloud.stream.bindings.output.destination=ticktock`, and `log-sink` will set
-`spring.cloud.stream.bindings.input.destination=ticktock`.
+.Spring Cloud Stream Application
+image::SCSt-with-binder.png[width=300,scaledwidth="50%"]
 
-==== Consumer Group Support
+Spring Cloud Stream uses Spring Boot for configuration, and the binder abstraction makes it possible for Spring Cloud Stream applications to be flexible in terms of how it connects to the middleware.
 
-Spring Cloud Stream is a library focusing on building message-driven microservices, and more specifically stream
-processing applications. In such scenarios, communication between different logical applications follows a
-publish-subscribe pattern, with data being broadcast through a shared topic, but at the same time, it is important to
-be able to scale up by creating multiple instances of a given application, which are in a competing consumer
-relationship with each other.
+This can mean, for example, dynamically choosing the destinations that these channels connect to at runtime (e.g. Kafka topics or Rabbit MQ exchanges), via external configuration properties in any form that is supported by Spring Boot (application arguments, environment variables, `application.yml` files, etc). Taking the source example from the previous section, providing the `spring.cloud.stream.bindings.output.destination=raw-sensor-data` property to the application will cause it to write to the `raw-sensor-data` Kafka topic, or the `orders` exchange in Rabbit.
 
-Spring Cloud Stream models this behavior through the concept of a consumer group, which is similar to the notion of
-consumer groups in Kafka. Each consumer binding can specify a group name such as
-`spring.cloud.stream.bindings.input.group=foo` (the actual name of the binding may vary). Each consumer group bound to
-a given destination will receive a copy of the published data, but within the group, only one application will receive
-each specific message.
+On the other hand, since the application is not middleware-specific, Spring Cloud Stream will automatically detect and use a binder that is found on the classpath, so you can easily use different types of middleware with the same code, just by including a different binder at build time. For more complex use cases, Spring Cloud Stream also provides the ability of packaging multiple binders within the same application and choosing what type of binder should be used at runtime, and even if multiple binders should be used at runtime for different channels.
 
-If no consumer group is specified for a given binding, then the binding is treated as if belonging to an anonymous,
-independent, single-member consumer group. Otherwise said, if no consumer group is specified for a binding, it will be
-in a publish-subscribe relationship with any other consumer groups.
+=== Persistent publish subscribe and consumer groups
```

Comment:
Replace all references of "publish subscribe" --> "publish-subscribe" or "publish/subscribe"